content/source/docs/cloud/sentinel/import/tfstate.html.md

```diff
@@ -447,7 +447,7 @@ This namespace is indexed by output name.
 The `sensitive` value within the [output namespace](#namespace-outputs) is
 `true` when the output has been [marked as sensitive][ref-tf-sensitive-outputs].
 
-[ref-tf-sensitive-outputs]: /docs/configuration/outputs.html#sensitive-outputs
+[ref-tf-sensitive-outputs]: /docs/configuration/outputs.html#sensitive
```

Comment:
Ah! So, with that one, I ended up adding an invisible anchor link in the corresponding terraform core PR (https://github.com/hashicorp/terraform/pull/27307), because it seemed like being able to link to #sensitive was a win. But I'll add that suggestion anyway, why not.

Reply:
Nevermind! it seems like you made this change in the terraform repo :) 

![image](https://user-images.githubusercontent.com/7556325/102533355-2b24d300-405a-11eb-9c44-b2f8a4a60492.png)